### ISSUE TITLE
ctps values are wrong

### ISSUE THREAD

**Issue by Thoralf-M** [iotaledger/hornet#592]
RC7
Looks like the wrong ctps value is related to the incoming transactions
![image](https://user-images.githubusercontent.com/46689931/89464445-c9ca2a80-d770-11ea-98ba-47d12863f324.png)
![image](https://user-images.githubusercontent.com/46689931/89464872-73112080-d771-11ea-8bb7-0a76b3669c92.png)

**Comment by Thoralf-M** [comment]
```
2020-08-05T23:16:12+02:00       INFO    Tangle  Milestone confirmed (658003): txsConfirmed: 42988, txsValue: 0, txsZeroValue: 42988, txsConflicting: 0, collect: 129ms, total: 306ms
2020-08-05T23:16:12+02:00       INFO    Tangle  New solid milestone: 658003, 272.10 TPS, 4298.80 CTPS, 1579.86% conf.rate
2020-08-05T23:17:32+02:00       INFO    Tangle  Milestone confirmed (658011): txsConfirmed: 2398, txsValue: 0, txsZeroValue: 2398, txsConflicting: 0, collect: 101ms, total: 209ms                                                                       
2020-08-05T23:17:32+02:00       INFO    Tangle  New solid milestone: 658011, 239.90 TPS, 239.80 CTPS, 99.96% conf.rate
2020-08-05T23:17:42+02:00       INFO    Tangle  Milestone confirmed (658012): txsConfirmed: 2423, txsValue: 0, txsZeroValue: 2423, txsConflicting: 0, collect: 58ms, total: 173ms                                                                        
2020-08-05T23:17:42+02:00       INFO    Tangle  New solid milestone: 658012, 243.50 TPS, 242.30 CTPS, 99.51% conf.rate
2020-08-05T23:17:52+02:00       INFO    Tangle  Milestone confirmed (658013): txsConfirmed: 2387, txsValue: 0, txsZeroValue: 2387, txsConflicting: 0, collect: 64ms, total: 291ms
2020-08-05T23:17:52+02:00       INFO    Tangle  New solid milestone: 658013, 242.40 TPS, 238.70 CTPS, 98.47% conf.rate
```

**Comment by muXxer** [comment]
fixed with #593
